test/data/command.create.yaml

```diff
@@ -4,7 +4,14 @@ name: bar
 version: '1.0'
 description: |
   Lorem ipsum dolor sit amet.
-arguments: '-d <domain> -h <host>'
+arguments: '<OPTION> <TARGET>
```

Comment:
Do we need `|` to keep new lines?

Reply:
Newlines are not respected by the UI, so I always just assumed we used it for easier readability in the yaml file.

Reply:
Sorry I just realized you meant I should add `|`
Yes, you need to add `|` for new lines.